include/csc232.h

```diff
@@ -58,30 +58,42 @@ using std::setprecision;
 using std::setw;
 
 // TODO: 1.1 Declare a named constant ROSTER_SIZE initialized to the integer value 10.
-
+const int ROSTER_SIZE{5};
 
 // TODO: 1.3 Declare a named constant NUM_COURSES initialized to the integer value of 3.
-
+const int NUM_COURSES{3};
 
 // TODO: 1.2 Declare an array named as specified whose size is ROSTER_SIZE that stores floating-point numbers (e.g., double)
-
+double grades[ROSTER_SIZE];
 
 // TODO: 1.4 Declare a two-dimensional array as specified that has NUM_COURSES rows and ROSTER_SIZE columns that
 //  stores floating point numbers (e.g., double)
-
+double gradeBook[NUM_COURSES][ROSTER_SIZE];
 
 // TODO: 2.1 Declare a function named as specified that specifies two parameters. The first parameter is an integer that
 //  specifies the number of elements in the array to process and the second parameter is an array of doubles.
-
+void maxGrade(arr grades, int ROSTER_SIZE);
```

Comment:
PROGRAM CORRECTNESS: You're not declaring the array parameter correctly. This is preventing the test target from building as this file won't compile.